x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/data_table/last_updated.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import styled from 'styled-components';
+
+import { useEventDetailsWidthContext } from '../../../../../common/components/events_viewer/event_details_width_context';
+import { useKibana } from '../../../../../common/lib/kibana';
+
+export const isCompactFooter = (width: number): boolean => width < 600;
+
+interface FixedWidthLastUpdatedContainerProps {
+  updatedAt: number;
+}
+
+export const FixedWidthLastUpdatedContainer = React.memo<FixedWidthLastUpdatedContainerProps>(
```

Comment:
Reminder: need to check why is this needed when there is a last updated component already.

Reply:
@YulNaumenko  do you remember?

Reply:
Separated out and deduped last updated component. This is the final one with tests : https://github.com/elastic/kibana/pull/176064/files#diff-36743070b5744453188c3858a3869fa054832cba119926b371108f818cb01a77

Reply:
Here is the commit : 4c8fe9158be41febba0b179242d32ef260905d9d